.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,55 @@
+name: pull request
+
+on: [pull_request, workflow_dispatch]
+
+jobs:
+  code_checks:
+    name: Code checks
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+      - name: Set Node Version
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.14.2
+      - name: Install deps
+        run: npm ci
+      - name: Code Checks
+        run: npm run prepublishOnly
+
+  publish_npm_betas:
+    name: Publish NPM beta versions
+    runs-on: ubuntu-latest
+    needs: [check_fork]
```

Comment:
I think this should be changed to `code_checks`, along with any other references using `check_fork`.

Reply:
good catch!